packages/workflow/src/ObservableObject.ts

```diff
@@ -1,20 +1,28 @@
+/** @format */
 
-import {
-	IDataObject,
-	IObservableObject,
-} from './';
+import { IDataObject, IObservableObject } from './';
 
 export interface IObservableOptions {
 	ignoreEmptyOnFirstChild?: boolean;
 }
 
-export function create(target: IDataObject, parent?: IObservableObject, option?: IObservableOptions, depth?: number): IDataObject {
-	depth = depth || 0;
+export function create(
+	target: IDataObject,
+	parent?: IObservableObject,
+	option?: IObservableOptions,
+	depth?: number,
+): IDataObject {
+	depth = depth ?? 0;
```

Comment:
i think this is the same
```suggestion
	depth = 0,
): IDataObject {
```

Reply:
Have you considered the `no-param-reassign` eslint rule? I found it helps to aid clarity. though that might conflict with default values for arguments like above